code/modules/mob/mob_transformation_simple.dm

```diff
@@ -66,6 +66,7 @@
 	else
 		desired_mob.key = key
 
+	SEND_SIGNAL(src, COMSIG_LIVING_ON_WABBAJACKED)
```

Comment:
This should send its own signal, especially because it is not the same as the wabbajack proc.